packages/documentsearch/src/searchview.tsx

```diff
@@ -270,15 +272,34 @@ interface IUpDownProps {
   trans: TranslationBundle;
 }
 
+
+
+
+// Getting the CommandRegistry object
+const commands = new CommandRegistry();
```

Comment:
We must use the command registry of the all application. This is the tricky part to fix the issue. The application command registry can be obtained in a plugin from the `app` object using `app.commands`. And to pass it in this part of the code, I would advice you to pass it as new argument of  `SearchDocumentView` at:

https://github.com/jupyterlab/jupyterlab/blob/19050c7b816bb4dd412a4bed94aee7a34050e5f4/packages/documentsearch-extension/src/index.ts#L162

Then you will be able to pass it through the react component tree to use it in `UpDownButtons`.